server_addon/substancepainter/client/ayon_substancepainter/plugins/create/create_textures.py

```diff
@@ -234,9 +244,15 @@ def get_instance_attr_defs(self):
     def get_pre_create_attr_defs(self):
         # Use same attributes as for instance attributes
         attr_defs = []
-        if  substance_painter.application.version_info()[0] >= 10:
+        if substance_painter.application.version_info()[0] >= 10:
             attr_defs.append(
                 BoolDef("use_selection", label="Use selection",
-                        tooltip="Select Layer Stack(s) for exporting")
+                        tooltip="Select Layer Stack(s) for exporting"),
+
             )
+
+        attr_defs.append(BoolDef("follow_workfile_version",
+                                 label="Follow Workfile Version",
+                                 default=self.follow_workfile_version))
```

Comment:
Out of curiosity - why are we making this configurable by the artist? Usually the studio define whether it's one OR the other and not up to the artist to pick and choose.